rebuildconda

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# Setting up conda:
+# Pass in the version of python as an argument with default of 2.7
+
+##########  Current setup BEGIN:
+_PYVER=${1:-2.7}
+#_PYVER=2.7
+#_PYVER=3.4
+#_PYVER=3.6
+penv_version=py${_PYVER}
+echo rebuilding conda environment $penv_version
+
+# List envs:
+conda info --envs
+# List all info:
+conda info -a
+
+source deactivate
+conda env remove --yes --name $penv_version
+conda create --yes --name $penv_version python=${_PYVER} ; source activate $penv_version
+conda config --get channels
+conda config --add channels conda-forge
+conda config --get channels
+# wip add packages that should be in base:
+base_packages1=" cython scipy scikit-learn pandas joblib nose typing "
+base_packages2=" jupyter "
+base_packages21=" jupyter_contrib_nbextensions jupyter_nbextensions_configurator "
+base_packages3=" boto boto3 "
```

Comment:
These should be deleted - they are part of my personal base set of packages that came over when I initialized this script.